src/Features/BannedSymbols.txt

```diff
@@ -0,0 +1,4 @@
+T:Microsoft.CodeAnalysis.CodeStyle.CodeStyleOption`1; Use 'Microsoft.CodeAnalysis.CodeStyle.CodeStyleOption2' instead
```

Comment:
Banned APIs to ensure our analyzer and fixer at least stick to just the internal option types. This will ease move of analyzers to shared layer, and also keep the entire project on a single API set. I think we should add these to different layers above from here as well.

Reply:
You also added this to **src/Features/Directory.Build.props**?

Reply:
Yes, that is correct - I also validated the diagnostics fire in Features layer, which I used to move all IDE analyzers to new Options API.

Reply:
Should this also be added for higher layers as well, since you did at least some effort to move them?